pkg/controller/certificaterequest/certificaterequest_controller_test.go

```diff
@@ -210,16 +211,20 @@ func TestReconcile(t *testing.T) {
 			expectError: false,
 		},
 		{
-			name: "discover clusterdeployment if certrequest ownerref is missing",
+			name: "reqeusts a new cert",
 			clientObjects: func() []runtime.Object {
 				cr := &certmanv1alpha1.CertificateRequest{}
 				cr.TypeMeta = certRequest.TypeMeta
 				cr.ObjectMeta = certRequest.ObjectMeta
-				cr.ObjectMeta.OwnerReferences = []metav1.OwnerReference{}
+				//cr.ObjectMeta.OwnerReferences = []metav1.OwnerReference{} // this should already be set to actual owner refs by ^
```

Comment:
I'm not exactly following this comment. Are you saying ObjectMeta should already have the OwnerReferences?

Reply:
yeah, the line above (217) sets `cr.ObjectMeta.OwnerReference` when `cr.ObjectMeta` is set